core/src/main/java/com/hcl/ecomm/core/utility/ProductUtility.java

```diff
@@ -0,0 +1,88 @@
+package com.hcl.ecomm.core.utility;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+
+public class ProductUtility {
+
+	private static final Logger LOG = LoggerFactory.getLogger(ProductUtility.class);
+
+	public static JsonArray fromStringToJsonArray(String responseStream) {
+
+		JsonArray j_array;
+		JsonObject jobj = new Gson().fromJson(responseStream, JsonObject.class);
+		if (jobj.has("items")) {
+			j_array = jobj.getAsJsonArray("items");
+		} else {
+			j_array = jobj.getAsJsonArray();
+		}
+		return j_array;
+	}
+
+	public static Map<String, String> fromJsonArrayToMap(JsonArray productJsonArray) {
+
+		HashMap<String, String> productMap = new HashMap<>();
+
+		try {
+
+			JsonObject productObject = productJsonArray.get(0).getAsJsonObject();
+			if (!productObject.isJsonNull()) {
+				Set<String> keys = productObject.keySet();
+				productMap.put("description", getDescription(productObject));
+				productMap.put("categoryId", getProductCategoryId(productObject).toString());
+				for (String key : keys) {
+					productMap.put(key, productObject.get(key).getAsString());
+				}
+			}
+		} catch (Exception e) {
+			LOG.error("Caught exception : " + e.getMessage());
+			e.printStackTrace();
```

Comment:
@nitheshkumar-hcl   Why you kept e.printStackTrace().? pl remove.

Reply:
Removed the stack trace